<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QWidget" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Please choose your class title:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>PIC10A</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PIC10B</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>301</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QSlider" name="horizontalSlider_3">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>HW4</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>HW5</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSlider" name="horizontalSlider">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSlider" name="horizontalSlider_2">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSlider" name="horizontalSlider_5">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>HW2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>HW3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>HW1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSlider" name="horizontalSlider_4">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSpinBox" name="spinBox">
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSpinBox" name="spinBox_2">
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="spinBox_3">
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QSpinBox" name="spinBox_4">
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QSpinBox" name="spinBox_5">
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>301</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Final</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Midterm 1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Midterm 2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSlider" name="horizontalSlider_7">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSlider" name="horizontalSlider_8">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSlider" name="horizontalSlider_6">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSpinBox" name="spinBox_6">
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSpinBox" name="spinBox_7">
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="spinBox_8">
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Grading Scheme:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>Scheme A</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>Scheme B</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>332</x>
     <y>280</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>CALCULATE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>492</x>
     <y>280</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Digital-7</family>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>597</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_7</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_7</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>525</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_8</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_8</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>529</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>576</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_7</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_7</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>595</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>548</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_8</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_8</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>592</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
